dist/redhat/scylla.spec.mustache

```diff
@@ -7,7 +7,7 @@ Group:          Applications/Databases
 License:        AGPLv3
```

Comment:
Please file an issue in the scylla.git repo and reference it instead of the enterprise issue.

Reply:
Created issue #5639, pushed new commit message https://github.com/scylladb/scylla/pull/5599/commits/e689183e54775e671950c750341bf4b60915e87f